docs/release-notes/8.6.md

```diff
@@ -0,0 +1,236 @@
+- hostapd
+- libgcab1-devel
+- libguestfs-appliance
+- libnbd-bash-completion
+- libpinyin-devel
+- libqrtr-glib
+- libselinux-static
+- libss-devel
+- libtpms
+- libtpms-devel
+- libuser-devel
+- libvirt-wireshark
+- libxmlb-devel
+- lmdb-devel
+- maven-openjdk17
+- nbdkit-gzip-filter
+- nbdkit-nbd-plugin
+- nbdkit-tar-filter
+- nbdkit-tar-plugin
+- nbdkit-tmpdisk-plugin
+- netavark
+- NetworkManager-initscripts-updown
+- nftables-devel
+- nginx-mod-devel
+- nmstate-devel
+- nmstate-libs
+- osbuild-composer-dnf-json
+- osbuild-luks2
+- osbuild-lvm2
+- pcp-pmda-denki
+- pcp-pmda-mongodb
+- php-pecl-xdebug3
+- multiple perl-\* that are in perl 5.32 modules
+- prometheus-jmx-exporter-openjdk11
+- python38-resolvelib
+- python3-bind9.16
+- python3-dnf-plugin-modulesync
+- qatlib-tests
+- qatzip
+- qatzip-devel
+- qatzip-libs
+- qemu-kvm-docs
+- qemu-kvm-hw-usbredir
+- qemu-kvm-ui-opengl
+- qemu-kvm-ui-spice
+- rig
+- rsyslog-mmfields
+- rust-std-static-wasm32-wasi
+- shadow-utils-subid
+- shadow-utils-subid-devel
+- sshpass
+- stress-ng
+- swtpm
+- swtpm-devel
+- swtpm-libs
+- swtpm-tools
+- swtpm-tools-pkcs11
+- virt-v2v-bash-completion
+- virt-v2v-man-pages-ja
+- virt-v2v-man-pages-uk
+- webrtc-audio-processing-devel
+- weldr-client
+
+
+## Branded packages
+
+Multiple packages have names changed due to branding, including:
+
+- redhat-backgrounds -> el-backgrounds
+- redhat-indexhtml -> el-indexhtml
+- redhat-logos -> el-logos
+- redhat-logos-httpd -> el-logos-httpd
+- redhat-logos-ipa -> el-logos-ipa
+- redhat-release -> el-release
+- redhat-release-eula -> el-release - we include EULA in release package
+
+All these packages have proper RPM `provides` declaration.
+
+The list of the packages that are changed due to branding includes, among the
+others:
+
+- abrt
+- anaconda
+- anaconda-user-help
+- cloud-init
+- crash
+- firefox
+- httpd
+- initial-setup
+- ipa
+- libguestfs
+- libreoffice
+- libreport
+- nginx
+- opa-ff
+- opa-fm
+- openscap
+- osinfo-db
+- python3
+- redhat-rpm-config
+- scap-security-guide
+- skopeo
+- sos
+- subscription-manager
+- thunderbird
+
+## Removed packages
+
+The following packages were removed:
+
+- rhc and related packages 
+- insights-client and related packages
+
+## Batteries
+
+Batteries repository contains packages that are needed to rebuild the
+distribution but are not included in any upstream repositories. Right now,
+this repository includes more than 2500 packages for each buildroot.
+
+Multiple batteries were updated for this release including:
+
+- Bootstraps for Perl 5.32
+- Updated mesa batteries
+- Smaller packages that broke buildroot
+
+
+About 420 packages were updated for x86_64 buildroot alone.
+
+
+## Gaia build system
+
+We are still working on Gaia 3.0.
+
+## Additional resources
+
+- [Download EuroLinux ISO](https://fbi.cdn.euro-linux.com/isos/)
+- [EuroLinux Public Request for Change and Bug Tracker](https://github.com/EuroLinux/eurolinux-distro-bugs-and-rfc)
+- [Red Hat 8.6 Beta
```

Comment:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.6_release_notes/index

Reply:
Change link and description.